10_Download Time Calculator.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+
+# Write a procedure download_time which takes as inputs a file size, the
+# units that file size is given in, bandwidth and the units for
+# bandwidth (excluding per second) and returns the time taken to download 
+# the file.
+# Your answer should be a string in the form
+# "<number> hours, <number> minutes, <number> seconds"
+
+# Some information you might find useful is the number of bits
+# in kilobits (kb), kilobytes (kB), megabits (Mb), megabytes (MB),
+# gigabits (Gb), gigabytes (GB) and terabits (Tb), terabytes (TB).
+
+# print 2 ** 10      # one kilobit, kb
+# print 2 ** 10 * 8  # one kilobyte, kB
+
+# print 2 ** 20      # one megabit, Mb
+# print 2 ** 20 * 8  # one megabyte, MB
+
+# print 2 ** 30      # one gigabit, Gb
+# print 2 ** 30 * 8  # one gigabyte, GB
+
+# print 2 ** 40      # one terabit, Tb
+# print 2 ** 40 * 8  # one terabyte, TB
+
+# Often bandwidth is given in megabits (Mb) per second whereas file size 
+# is given in megabytes (MB).
+
+def convert_seconds(number):
+    time = ''
+    hrs = int(number / 3600)
+    mins = int((number - hrs * 3600) / 60)
+    secs = int(number - hrs * 3600 - mins * 60)
+    if hrs == 1:
+        time = time + str(hrs) + ' hour '
+    else:
+        time = time + str(hrs) + ' hours '
+    if mins == 1:
+        time = time + str(mins) + ' minute '
+    else:
+        time = time + str(mins) + ' minutes '
+    if secs == 1:
+        time = time + str(secs) + ' second '
+    else:
+        time = time + str(secs) + ' seconds '

```

Comment:
we can simplify these conditions as below

`time = time + str(secs) + ' second ' if secs == 1 else ' seconds'`

Reply:
done